### ISSUE TITLE
cannot add ssh key

### ISSUE THREAD

**Issue by tillmo** [ontohub/ontohub#1661]
This is really annoying, because I want to use ontohub productively.
```
Errno::EACCES (Permission denied @ rb_sysopen - /data/git/.ssh/authorized_keys):
  lib/authorized_keys_manager.rb:60:in `initialize'
  lib/authorized_keys_manager.rb:60:in `open'
  lib/authorized_keys_manager.rb:60:in `in_authorized_keys'
  lib/authorized_keys_manager.rb:24:in `add'
  app/models/key.rb:26:in `add_to_authorized_keys'
```

**Comment by eugenk** [comment]
This is based on #1646. For some reason, puma runs as user `ontohub`, but it needs to run as user `webservd`. `ontohub` has no permission to change the `authorized_keys`.

**Comment by eugenk** [comment]
Maybe this is a result of the conflict that shall be solved with #1538.

**Comment by eugenk** [comment]
This is fixed for now, but #1646 is really important for this (and several other issues) not to occur again.
